docs/sign-multisig.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Sign a Multisig transaction with the Flow CLI
+sidebar_title: Sign a Multisig Transaction
+description: How to sign a Multisig transaction from the command line
+---
+
+The Flow CLI provides a command to sign a multsi transaction with options to specify
+signer account.
+
+Use this functionality after a multisig transaction has been generated:
+ - When a multisig transaction the keys needed to sign have already been identified
+ - This signing method will first pull down the transaction RLP from url to allow the user to sign
+ - After the transaction has been signed the resulting signed RLP will be posted to the server
+
+```shell
+flow multisig sign <transaction id ulr>
```

Comment:
What was the thought process behind adding a new sub command, rather than having it as part of the `sign` sub command, and just signalling via a flag, i.e. `--remote-url` or similar?

Reply:
My thinking was "multisig" is a new concept that might grow so wanted to keep it separate from existing commands.